app/views/metrics/index.html.erb

```diff
@@ -232,9 +232,24 @@
         </span>
       </div>
     </div>
+    <br>
+
+    <div class="metric">
+      <div class="inline field">
+        <div class="ui checkbox conditions" id='extension_requests_checkbox'>
+          <input type="checkbox" tabindex="0" class="hidden" name="metrics-checkbox-5">
+        </div>
+        Students who have greater than or equal to
+        <span class="padded-span">
+          <input type="text" placeholder="1" id='extension_requests_count'>
+        </span>
+        extensions.
+      </div>
+    </div>
+
     <br>
     <% if @cud.instructor? %>
-      <button class="ui disabled submit button" id="save">SAVE</button>
```

Comment:
Why are you removing `disabled`?

In metrics.js, it is removed if there was a change to the form.
![Screenshot 2023-04-16 at 15 19 14](https://user-images.githubusercontent.com/9074856/232336827-d4dd4543-6856-4dce-ac4b-0af48185547c.png)

Reply:
I did this because, the default now is that when you enter into metrics, a checkbox is marked but the save button is disabled even when you fill in the field, which is quite non-intuitive. But now thinking about it the solution should be to not have any boxes checked initially instead of removing the disabled class.